src/matrixCells/makeMatrixArgs.js

```diff
@@ -126,8 +126,8 @@ export default function make_matrix_args(regl, store) {
       zoom: zoom_function,
       interp_uni: (ctx, props) => Math.max(0, Math.min(1, props.interp_prop)),
       run_animation: regl.prop("run_animation"),
-      pos_rgb: store.select("cat_viz").mat_colors.pos_rgb,
-      neg_rgb: store.select("cat_viz").mat_colors.neg_rgb,
+      pos_rgb: store.select("cat_viz").mat_colors.pos,
+      neg_rgb: store.select("cat_viz").mat_colors.neg,
```

Comment:
Just curious what's the motivation for changing the variable name here?
It seems to go against what Clustergrammer held previously.

Reply:
The reasoning was originally that I was going to allow for hex strings or rgb(a) strings or arrays. However because this goes into regl it has to be a RGB array. Reverted.